src/cmr_notifier/main.py

```diff
@@ -1,11 +1,15 @@
 import datetime
 import json
 import os
+import urllib.parse
 
 import boto3
 import requests
 
 
+CMR_SEARCH_URL = os.environ['CMR_SEARCH_URL']
+CMR_PROVIDER = os.environ['CMR_PROVIDER']
```

Comment:
I find the module is generally easier to test if these are retrieved in `lambda_handler` and passed down as function parameters to where they're needed, similar to how `TOPIC_ARN` and `TABLE_NAME` are currently handled.